packages/blade/src/components/List/listTokens.ts

```diff
@@ -29,31 +31,25 @@ const listItemMarginLeft: Record<
   },
 };
 
-const listItemBulletMarginRight: Record<
-  NonNullable<ListProps['variant']>,
-  DotNotationSpacingStringToken
```

Comment:
is `DotNotationSpacingStringToken` useful at all now? or basically the `SpacingValueType` is same as `DotNotationSpacingStringToken`? 🤔

Reply:
```ts
type SpacingValueType = DotNotationSpacingStringToken | `${string}${SpaceUnits}` | 'auto';
```

This is spacing value type. So its DotNotationSpacingStringToken + absolute values